### ISSUE TITLE
VNC remote connection crashes if I try to resize windows

### ISSUE THREAD

**Issue by carlossanlop** [linuxmint/cinnamon#9507]
* _Cinnamon version (cinnamon --version)_
    **4.0.10**

   - _Please specify if you are using the daily builds PPA_
     **No, I'm not**

 * _Distribution - (Mint 17.2, Arch, Fedora 25, etc...)_
    **Linux Mint 19.1**

 * _Graphics hardware *and* driver used_
    ```
    $ lspci | grep VGA
    00:02.0 VGA compatible controller: Intel Corporation Skylake GT2 [HD Graphics 520] (rev 07)
    ```

 * 32 or 64 bit
    **64 bit**

 * Attach /home/<username>/.xsession-errors, or /var/log/syslog

Here are the relevant messages in /var/log/syslog from the moment I create the VNC connection, to the point where VNC crashes:

```
Aug  5 19:47:14 myhostname x11vnc[16285]: PORT=5900
Aug  5 19:47:14 myhostname systemd[1]: Started VNC Server for X11.
Aug  5 19:48:48 myhostname rtkit-daemon[1458]: Supervising 3 threads of 1 processes of 1 users.
Aug  5 19:48:48 myhostname rtkit-daemon[1458]: message repeated 3 times: [ Supervising 3 threads of 1 processes of 1 users.]
Aug  5 19:48:48 myhostname rtkit-daemon[1458]: Successfully made thread 16680 of process 16551 (n/a) owned by '1000' RT at priority 10.
Aug  5 19:48:48 myhostname  rtkit-daemon[1458]: Supervising 4 threads of 2 processes of 1 users.
Aug  5 19:48:50 myhostname rtkit-daemon[1458]: message repeated 6 times: [ Supervising 4 threads of 2 processes of 1 users.]
Aug  5 19:51:53 myhostname systemd[1]: x11vnc.service: Main process exited, code=exited, status=2/INVALIDARGUMENT
Aug  5 19:51:53 myhostname systemd[1]: x11vnc.service: Failed with result 'exit-code'.
Aug  5 19:51:53 myhostname systemd[1]: x11vnc.service: Service hold-off time over, scheduling restart.
Aug  5 19:51:53 myhostname systemd[1]: x11vnc.service: Scheduled restart job, restart counter is at 10.
Aug  5 19:51:53 myhostname systemd[1]: Stopped VNC Server for X11.
```

**Issue**

If I remotely connect to my Linux Mint machine using VNC, then try to resize any window, the VNC connection crashes.

My local machine has Windows 10 64 bits and I am using VNC Viewer 6.20.529 to connect to the Linux machine.

**Steps to reproduce**

Connect to the machine via VNC. The image quality and encryption are irrelevant.
Open Mousepad or the GNOME Terminal.
Try to resize the window.
The VNC connection is killed.

**Expected behaviour**

The window resizes without crashing the VNC connection.

**Comment by leigh123linux** [comment]
cinnamon is EOL for your version of mint and wont be fixed, can you reproduce with cinnamon-4.6.x in mint-20?

**Comment by carlossanlop** [comment]
I tried dist-upgrading to Mint 20 but my computer got in such a bad state at the end, I ended up formatting it 😢. I decided to try Pop!_OS this time.
If I get a chance to try Mint 20 again, I'll report the issue again.
Thanks anyway.
